src/tasks.md

```diff
@@ -57,30 +57,31 @@ You can list all the `kwctl` options and subcommands by running the following co
 
 ```shell
 $ kwctl --help
-kwctl 0.2.4
-Flavio Castelli <fcastelli@suse.com>:Rafael Fernández López <rfernandezlopez@suse.com>
+kwctl 0.2.5
+Kubewarden Developers <kubewarden@suse.de>
 Tool to manage Kubewarden policies
 
 USAGE:
-    kwctl [FLAGS] <SUBCOMMAND>
+    kwctl [OPTIONS] <SUBCOMMAND>
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+OPTIONS:
+    -h, --help       Print help information
     -v               Increase verbosity
+    -V, --version    Print version information
 
 SUBCOMMANDS:
-    annotate       Adds Kubewarden metadata to a WebAssembly module
-    completions    Generates shell completions
-    help           Prints this message or the help of the given subcommand(s)
-    inspect        Inspects Kubewarden policy
-    manifest       Scaffolds a Kubernetes resource
+    annotate       Add Kubewarden metadata to a WebAssembly module
+    completions    Generate shell completions
+    digest         Fetch the digest of its OCI manifest
+    help           Print this message or the help of the given subcommand(s)
+    inspect        Inspect Kubewarden policy
     policies       Lists all downloaded policies
     pull           Pulls a Kubewarden policy from a given URI
     push           Pushes a Kubewarden policy to an OCI registry
     rm             Removes a Kubewarden policy from the store
     run            Runs a Kubewarden policy from a given URI
-    verify         Verifies a Kubewarden policy from a given URI using Sigstore
+    scaffold       Scaffold a Kubernetes resource or configuration file
+    verify         Verify a Kubewarden policy from a given URI using Sigstore
```

Comment:
```suggestion
    verify         Verifies a Kubewarden policy from a given URI using [sigstore](https://github.com/sigstore)
```

Reply:
I misunderstood the assignment, sorry. We can just change the verb and not include the link.

Reply:
I would prefer to not change the verbs and leave them in imperative voice, as other cli utilities do (e.g: `git`).

Reply:
So, the rationale behind changing the voice was because this was the associated documentation. In the actual CLI utility, I don't recommend changing it.

Reply:
But, this is just a copy-paste from the command help output and stated as so in the docs, and we will refresh that output when we change commands in the future too (hence why I think it should match what a user gets when running the command help, and retouching it in the docs page is bound to always be out of sync with changes to the subcommand lists).